packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -240,21 +312,108 @@ export class Builder extends EventEmitter {
     }
   }
 
-  private async updateDocumentForFile(file: string) {
-    const contents = await readFile(file, 'utf8');
-    if (contents.trim().length === 0) {
-      return;
+  private async updateDocumentForFile(filePath: string) {
+    const project = this.config.getConfigForFile(filePath);
+
+    if (project) {
```

Comment:
i think this should always be true, so we may want to `error` on the else? technically our file paths should be coming from `includes` globs so they should always belong to a project. This was added because of the absolute path globbing issue.